src/xrEngine/defines.h

```diff
@@ -6,9 +6,12 @@ ENGINE_API extern bool bDebug;
 #define bDebug 0
 #endif
 
-extern ENGINE_API bool CallOfPripyatMode;
-extern ENGINE_API bool ClearSkyMode;
-extern ENGINE_API bool ShadowOfChernobylMode;
+enum
+{
+    callOfPripyatMode       = (1ul << 0ul),
+    clearSkyMode            = (1ul << 1ul),
+    shadowOfChernobylMode   = (1ul << 2ul),
```

Comment:
General code style is: global variables should be PascalCase.
```suggestion
    CallOfPripyatMode       = (1ul << 0ul),
    ClearSkyMode            = (1ul << 1ul),
    ShadowOfChernobylMode   = (1ul << 2ul),
```

Though, in fact, there are several code styles that can be used in the engine...
Sometimes, snake_case can be used, with `g_` or `s_` prefix to the variables.
Often, the current file code style is used.